versions/1/_dev/spec.yml

```diff
@@ -1,5 +1,6 @@
 spec:
   additionalContents: false
+  developmentFolder: true
```

Comment:
nit: `_dev` is also in data_stream. You may add a test case that verifies that one too.

Reply:
Good catch, for some reason I thought they shared the same spec. Fixed.